templates/ios/Example/PROJECT.xcodeproj/project.pbxproj

```diff
@@ -29,12 +29,12 @@
 			containerPortal = 6003F582195388D10070C39A /* Project object */;
 			proxyType = 1;
 			remoteGlobalIDString = 6003F589195388D20070C39A;
-			remoteInfo = "PROJECT";
+			remoteInfo = PROJECT;
```

Comment:
Thanks!

So a few of these changes will break when people use spaces in their pod names, which they do. Any chance you can add some quotes around these please? There's one or two down below too.